de/frames_rover/aion_r1.md

```diff
@@ -0,0 +1,70 @@
+# Aion Robotics R1 UGV
```

Comment:
@PerFrivik Any changes in this /de path are the german files. Can you please remove from this path and move to the /en path (translations are round tripped through crowdin from /en to the other paths.

Also, there flaw checker says some images aren't linked. If they aren't going to be used, please remove them.